features/camel/src/main/feature/feature.xml

```diff
@@ -119,6 +119,7 @@
         <bundle>mvn:io.netty/netty-transport-native-epoll/${netty41-version}
         </bundle>
         <bundle>mvn:io.netty/netty-transport-native-kqueue/${netty.version}</bundle>
+        <bundle>mvn:io.netty/netty-transport-native-unix-common/${netty.version}</bundle>
```

Comment:
This change here concerns me a small bit. The netty upgrade split out some package, clearly, but I'm unclear why the unix-common bundle really needed to be added, particularly as I was doing my development on OSX. This change, right here, is one of the most important ones to be tested across OSs.